tests/test_gin.py

```diff
@@ -91,7 +92,7 @@ class TestNwbConversions(unittest.TestCase):
 
         @parameterized.expand(parameterized_recording_list)
         def test_convert_recording_extractor_to_nwb(self, recording_interface, interface_kwargs):
-            nwbfile_path = self.savedir / f"{recording_interface.__name__}.nwb"
+            nwbfile_path = str(self.savedir / f"{recording_interface.__name__}.nwb")
```

Comment:
Worth mentioning; this change doesn't actually effect the CI, but the string casting seems important for running these tests on a local device (may be system specific) with slightly different versions of upstream packages than the CI installs.